src/overlays/actors/ovl_Obj_Etcetera/z_obj_etcetera.c

```diff
@@ -163,20 +164,24 @@ void ObjEtcetera_Idle(ObjEtcetera* this, PlayState* play) {
                 // start of launching
                 this->oscillationTimer = 10;
             }
+
             this->burrowFlag |= 1;
         } else {
             if (this->burrowFlag & 1) {
                 // Player is walking off the Deku Flower
                 this->oscillationTimer = 10;
                 ObjEtcetera_StartRustleAnimation(this);
             }
+
             this->burrowFlag &= ~1;
```

Comment:
I suppose this could have a macro, but if there's only one there's not much point.